src/table/table-role/grid-navigation.tsx

```diff
@@ -244,49 +249,43 @@ class GridNavigationProcessor {
 
   private updateFocusTarget() {
     this.focusTarget = this.getSingleFocusable();
-    this.focusables.forEach(focusable => {
-      const element = focusable.current;
-      const handler = this.focusHandlers.get(focusable)!;
-      handler(this.focusTarget, this.isSuppressed(element));
-    });
+    for (const [focusable, prevIsFocusable] of this.focusables) {
```

Comment:
When I first read this I thought `focusable` and `prevIsFocusable` were both boolean and the latter was keeping the value of the previous render (similar to using our `usePrevious` hook). What about some minor renaming, like

```
for (const [focusableElement, isFocusable] of this.focusables) {
   const newIsFocusable = this.focusTarget === focusable || this.isSuppressed(focusable);
   if (newIsFocusable !== isFocusable) {
    ...
```